.github/workflows/auto_update_doc.yml

```diff
@@ -3,6 +3,9 @@ on:
   pull_request_target:
   workflow_dispatch:
 
+permissions:  # set top-level default permissions as security best practice
+  contents: read # TODO: Check https://github.com/ossf/scorecard practices
```

Comment:
This url seems wrong.

Reply:
thank you, you are write, it should be https://github.com/ossf/scorecard/blob/7ce8609469289d5f3b1bf5ee3122f42b4e3054fb/docs/checks.md#token-permissions I'll update that.